src/dodal/beamlines/training_rig.py

```diff
@@ -31,13 +29,18 @@
 set_log_beamline(BL)
 set_utils_beamline(BL)
 
-set_path_provider(
-    StaticVisitPathProvider(
-        BL,
-        Path("/exports/mybeamline/data/2025"),
-        client=LocalDirectoryServiceClient(),
-    )
-)
+# Leave the old path provider so it can be uncommented and used in the meantime while
+# we transition
+
+# set_path_provider(
+#     StaticVisitPathProvider(
+#         BL,
+#         Path("/exports/mybeamline/data/2025"),
+#         client=LocalDirectoryServiceClient(),
+#     )
+# )
+
+set_path_provider(StartDocumentBasedPathProvider())
```

Comment:
Are we decided on calling this from inside the dodal module or from blueapi?

Reply:
This gets called when blueapi loads the context.